lib/tuya.js

```diff
@@ -1199,7 +1220,7 @@ const valueConverterBasic = {
 
 const valueConverter = {
     trueFalse: valueConverterBasic.lookup({1: true, 0: false}),
-    onOff: valueConverterBasic.lookup({'ON': true, 'OFF': false}),
+    onOff: {from: (v) => v ? 'ON' : 'OFF', to: (v) => v === 'ON'},
```

Comment:
@Koenkk
Datapoint with `online` key sometimes reports data with an enum datatype, so the current version of the converter doesn't handle it. Here are the messages I sniffed:
`01007301000101`
`00527301000100`
`00347304000101`

Is it ok to modify this converter or is it better to create a new one specifically for this case?